packages/config/config/devices/0x0086/dsb09.json

```diff
@@ -18,908 +18,199 @@
 	},
 	"associations": {
 		"1": {
-			"label": "Report group 1",
+			"label": "Lifeline",
 			"maxNodes": 5,
 			"isLifeline": true
 		},
 		"2": {
-			"label": "Report group 2",
+			"label": "Group 2",
 			"maxNodes": 5,
 			"isLifeline": true
 		},
 		"3": {
-			"label": "Report group 3",
+			"label": "Group 3",
 			"maxNodes": 5,
 			"isLifeline": true
 		}
 	},
 	"paramInformation": {
 		"1": {
-			"label": "Voltage",
-			"description": "Voltage to calculate wattage from amperage.",
+			"label": "Operating Voltage",
 			"valueSize": 2,
-			"unit": "V",
-			"minValue": 0,
-			"maxValue": 32000,
+			"minValue": 120,
+			"maxValue": 240,
 			"defaultValue": 240,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "120V",
+					"value": 120
+				},
+				{
+					"label": "240V",
+					"value": 240
```

Comment:
There should be a space between values and their unit. Probably affects a bunch of other locations aswell.